### PR TITLE
data: Azeezkadiri

### PR DESCRIPTION
**Pull request by Azeezkadiri**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4901"></a>

### CODE REVIEW

File: data/Azeezkadiri.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Abdulazeez Kadiri",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend engineer | JavaScript | react |technical writer | open source contributor",
+  "avatar": "https://github.com/Azeezkadiri.png"
```
Comment (swagataroy30) [inline data/Azeezkadiri.json:6]:
```suggestion
```

File: data/Azeezkadiri.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Abdulazeez Kadiri",
+  "displayStatsPublic": true,
```
Comment (swagataroy30) [inline data/Azeezkadiri.json:3]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture and also `displayStatsPublic: true` is no longer required. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

File: data/Azeezkadiri.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    {
+      "name": "Follow me on linkedin",
+      "url": "https://www.linkedin.com/in/abdulazeez-kadiri-918a1b213/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on github",
+      "url": "https://github.com/Azeezkadiri",
+      "icon": "FaGithub"
+    },
```
Comment (swagataroy30) [inline data/Azeezkadiri.json:22]:
```suggestion
    }
```

File: data/Azeezkadiri.json (hunk 2 above)
Comment (imbeshat) [inline data/Azeezkadiri.json:3]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
You need to remove the comma from line 22. Thank You!

File: data/Azeezkadiri.json (hunk 1 above)
Comment (imbeshat) [inline data/Azeezkadiri.json:6]:
```suggestion
```

File: data/Azeezkadiri.json (hunk 3 above)
Comment (imbeshat) [inline data/Azeezkadiri.json:22]:
```suggestion
    }
```

**Review by imbeshat — CHANGES_REQUESTED** [review]
Please update these

File: data/Azeezkadiri.json (hunk 1 above)
Comment (Azeezkadiri) [reply data/Azeezkadiri.json:6]: Okay 👌🏾 thanks 🙏🏼

File: data/Azeezkadiri.json (hunk 2 above)
Comment (Azeezkadiri) [reply data/Azeezkadiri.json:3]: Copy

**Review by imbeshat — APPROVED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good! 👍 
Great collaboration everyone. 🎉

**Pull request merged**